client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -19,15 +20,22 @@ const Launchpad: Step = ( { navigation }: LaunchpadProps ) => {
 	const translate = useTranslate();
 	const almostReadyToLaunchText = translate( 'Almost ready to launch' );
 	const siteSlug = useSiteSlugParam();
-
+	const flow = useFlowParam();
 	const site = useSite();
 	const launchpadScreenOption = site?.options?.launchpad_screen;
 
 	useEffect( () => {
+		// launchpadScreenOption changes from undefined to either 'off' or 'full'
+		// we need to check if it's defined to avoid recording the same action twice
+		if ( launchpadScreenOption ) {
+			recordTracksEvent( 'calypso_launchpad_loaded', { flow: flow } );
+		}
+
 		if ( launchpadScreenOption === 'off' ) {
 			window.location.replace( `/home/${ siteSlug }/?forceLoadLaunchpadData=true` );
+			recordTracksEvent( 'calypso_launchpad_redirect_to_home', { flow: flow } );
 		}
```

Comment:
Right now, in the case that the launchpad will redirect this fires both `calypso_launchpad_loaded` and `calypso_launchpad_redirect_to_home`. While we can work with the events this way, it would probably be beneficial to make these one or the other.

There could be a few ways to set this up to handle this, but I think updating the first conditional to be `launchpadScreenOption && launchpadScreenOption !== 'off'` is one way to go about it.

Reply:
thanks Addie, updated